addons/binding/org.openhab.binding.neato/ESH-INF/thing/vacuumcleaner.xml

```diff
@@ -0,0 +1,243 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="neato" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+
+	<!-- DEFINITIONS of terms used in the binding: Neato is a XML document used for providing users with frequently updated content. -->
+
+	<!-- Neato Thing Type -->
+	<thing-type id="vacuumcleaner">
+		<label>Neato Vacuum Cleaner</label>
+		<description>Provides communication with Neato Vacuum cleaner.</description>
+
+		<channels>
+			<channel id="battery-level" typeId="battery-level" />
+			<channel id="state" typeId="state" />
+			<channel id="version" typeId="version" />
```

Comment:
It seems you did not use the standard properties yet for these, AFAIK these should even be properties instead of channels. So that at least applies to `version`, `model-name` and `firmware`.